baklava/package.json

```
@@ -51,7 +52,10 @@
     },
     "win": {
       "icon": "icons/icon.png",
-      "target": "msi"
+      "target": [
+        "msi",
+        "exe"
```

I'm slightly worried about keeping the msi target. In my fork, the working action uses only exe so that msi is not built. If msi is built then we are adding a working build to a failing action meaning it won't get published. Removing msi for now is probably best. We can add it back later if we fix it

```suggestion
        "exe"
```